drivers/flash/nrf_qspi_nor.c

```diff
@@ -673,9 +673,24 @@ static int qspi_nrfx_configure(const struct device *dev)
 	struct qspi_nor_data *dev_data = dev->data;
 	const struct qspi_nor_config *dev_config = dev->config;
 
+#if defined(BASE_CLOCK_DIV)
```

Comment:
I know that the `BASE_CLOCK_DIV` is currently only defined when `CONFIG_SOC_SERIES_NRF53X` is defined, but isn't it better to use `CONFIG_SOC_SERIES_NRF53X` here to notify that this code will be included only for the NRF53 series?

Reply:
Yes, this will be clearer. I'll update all occurrences of `#if defined(BASE_CLOCK_DIV)`.

Reply:
Done.